tests/scripts/test_tools_image_detect_changes.py

```
@@ -0,0 +1,97 @@
+        env.update(
+            {
+                "GITHUB_EVENT_NAME": "pull_request",
+                "GITHUB_OUTPUT": str(output_file),
+                "PR_BASE_SHA": "base-sha",
+                "PR_HEAD_SHA": "head-sha",
+                "MOCK_CHANGED_FILES": changed_files,
+                "PATH": f"{bin_dir}:{env['PATH']}",
+            },
+        )
+
+        result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)